Collapse.php

```diff
@@ -177,7 +181,13 @@ public function renderItem($header, $item, $index)
             if ($this->autoCloseItems) {
                 $headerOptions['data-parent'] = '#' . $this->options['id'];
             }
-            $headerToggle = Html::a($header, '#' . $id, $headerOptions) . "\n";
+            if($this->headerLink ===false){
+                $headerOptions= ArrayHelper::merge($headerOptions,['data-target'=>'#'.$id]);
```

Comment:
Why data-target?

Reply:
Please check the collapse component in bootstrap 3.2~. Data-target is another alternative method for collapse an object.

Reply:
Aha. OK.